doc/changelog.md

```diff
@@ -1,4 +1,18 @@
-# Release 0.6.0-dev
+# Release 0.7.0-dev
+
+<h3>New features</h3>
+
+<h3>Improvements</h3>
+
+<h3>Breaking changes</h3>
+
+<h3>Bug fixes</h3>
```

Comment:
I think this template is missing the internal changes header
```suggestion
<h3>Bug fixes</h3>

<h3>Internal changes</h3>

```

Reply:
I added it in a different commit.